typescript/tsconfig.node20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "lib": ["ES2023"],
+    "module": "node16",
+    "target": "ES2022",
+    "moduleResolution": "node16"
```

Comment:
Why does `node20-esm.json` use `NodeNext` but `node20.json` uses `node16`?

Reply:
that is the difference I believe between cjs and esm config

Reply:
oh unless they are the same result. Hm the typescript docs are kinda confusing

Reply:
`nodenext` is basically saying it could change behavior when you upgrade `typescript` (the same way `esnext` works)